doc/INITIALIZE.md

```diff
@@ -72,11 +71,108 @@ The odata-library currently support four authentication types.
 - basic - [use basic access authentication](https://en.wikipedia.org/wiki/Basic_access_authentication)
 - samlSap - specific authentication for sap based services
 - cookie - run authentication code externally and just pass authentication cookies to odata-library
+- cert - pass client certificate in PEM format
 
 If authentication type is not specified odata-library tries to use first three authentication
 types automatically.
 
-## Authenticate to service outside of Service
+### Client certificate authentication
+
+The client certificate settings are passed to https.Agent which is used
```

Comment:
missing space after dot ('https.Agent')

Reply:
Hi @jakub-vaclavik-sap ,

I am not understand. https.Agent is class from NodeJS.

https://nodejs.org/api/https.html#class-httpsagent

I have added the URL to the documentation.

Reply:
My bad, you are right.